app/src/test/java/com/findmyrecycling/ProductTests.kt

```diff
@@ -76,7 +76,7 @@ class ProductTests {
         assertTrue(allProducts!!.isNotEmpty())
         var containsCellPhones = false
         allProducts!!.forEach {
-            if (it.product.equals("Cell Phones")) {
+            if (it.product == "Cell Phones") {
```

Comment:
simplified operator

Reply:
This was covered in a previously submitted review. When that occurs, first in wins.